test/AutoDiff/validation-test/derivative_registration.swift

```diff
@@ -217,6 +217,7 @@ DerivativeRegistrationTests.testWithLeakChecking("DerivativeGenericSignature") {
   expectEqual(1000, dx)
 }
 
+#if REQUIRES_SRxxxx
```

Comment:
Rather than disabling on all platforms, could we disable only for the Windows platform? I know there's some controversy here that that may be an antipattern.

Reply:
We certainly could, though Im worried that there is a more delicate issue underlying these failures (it seems that the 4 failures are all caused by the same underlying cause related to casting).  I think that @rxwei was interested in looking at the issue, but I figure that this helps get the tests running earlier rather than later.

Reply:
I'm pretty sure the issues are related to the SIL differentiable function representation and reabstraction: something related to TF-851. This issue isn't caught in SIL verification because `@differentiable`-function-type-related instruction verification is also disabled: TF-1197.

The nature of the issue is pretty well understood, it should be fixable when it becomes a P0 for someone (I don't foresee personally prioritizing it).